test/params_api_test.c

```diff
@@ -664,11 +664,13 @@ static int test_param_modified(void)
 static int test_param_copy_null(void)
 {
     int ret, val;
-    int a = 1, b = 2;
+    int a = 1, b = 2, i= 0;
```

Comment:
Cosmetic nit - missing space before `=`

Reply:
I wish there were a few more emojiis to select from here.. :)

Reply:
There are :sparkles: for example [here](https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md)

:pirate_flag:

Reply:
:see_no_evil: